File: c/Lista1/Desafio3.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

main(){

    float cot, dolar, real;
    
    printf("Digite a cotação do dolar: ");
    scanf("%f", &cot);printf("Digite o valor em dolar: ");
    scanf("%f", &dolar);

	real = cot * dolar;
    printf("Valor em real é: %3.0f \n", real);
    
}

File: js/lista1/Desafio3.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

/*
Desafio3- Construa um programa que:

Leia a cotação do dólar;

Leia um valor em dólares;

Converta esse valor para Real;

Mostre o resultado.

Fórmula: Valor_Real = cot*valor_dolar.
*/

rl.question('Informe a cotação do dólar: ', (cot) => {
  rl.question('Informe o valor em dólares: ', (valor) => {

    const real = cot * valor;
    
    console.log(`O valor em reais é: ${real}`);
    rl.close();
  });
});
